types/foundry-vtt-api/types/core/keyboard.d.ts

```diff
@@ -0,0 +1,66 @@
+
+declare const KEYS: {
+	BACKSPACE: 8,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/foundry-vtt-api/types/core/keyboard.d.ts;
  Line: 3;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.